command/task_disable.go

```diff
@@ -101,3 +102,13 @@ func (c *taskDisableCommand) Run(args []string) int {
 
 	return ExitCodeOK
 }
+
+func processEOFError(scheme string, err error) error {
+	if strings.Contains(err.Error(), "EOF") && scheme == api.HttpScheme {
+		err = fmt.Errorf("%s. Scheme %s was used, "+
+			"this error can be caused by using http to connect to an https CTS instance, "+
+			"consider using %s scheme instead", err, scheme, api.HttpsScheme)
```

Comment:
> this error can be caused by a client using http to connect to a CTS server with TLS enabled, consider using https scheme instead

took a jab at rephrasing "an https CTS instance" since it reads colloquial. Feel free to revise as you see fit